Pester tests wouldn't run from build directory with spaces

BarryNolte opened the pull request:
Added quotes at the appropriate places to enable running commands that
contained spaces in the filename/path.  Also needed to add pester.psm1
to path of Import-Module.

msftclas commented:
Hi __@BarryNolte__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSBOT;

lzybkr commented on build.psm1:
```diff
@@ -567,8 +567,8 @@ function Start-PSPester {
 
-    $PesterModule = [IO.Path]::Combine((Split-Path $powershell), "Modules", "Pester")
-    $Command += "Import-Module $PesterModule; "
+    $PesterModule = [IO.Path]::Combine((Split-Path $powershell), "Modules", "Pester", "Pester.psm1")
```
Pester uses a module manifest, so importing the psm1 is not correct.

BarryNolte commented:
I updated the branch, I got ahead of myself on the fix and didn't need the hard coded file, thanks.

lzybkr commented:
@BarryNolte - to make reviewing easier on us, can you rebase or squash?

BarryNolte commented:
Updated, rebased and squashed.

BarryNolte committed:
Pester tests won't run from directory with space

Added quotes around file/pathnames as needed to get pester tests to run from build directory with a space in it.

msftclas commented:
__@BarryNolte__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, MSBOT;

dlwyatt commented:
Why is Invoke-Expression being used here?  Without it, you wouldn't have to worry about quoting anything manually.

lzybkr commented:
@dlwyatt - good call.  @mirichmo, who signed off on your change :)
